tox.ini

```diff
@@ -1,11 +1,31 @@
 [tox]
-envlist = py26,py27,py33
+envlist =
+  py27,
+  py34,
+  py35,
+  py36,
+  pypy,
+  pypy3,
+  coverage
 
 [testenv]
-commands = python setup.py test -q
-deps = zope.testrunner
-       zope.browserpage
-       zope.browserresource
-       zope.principalregistry
-       zope.publisher
-       zope.securitypolicy
+commands =
+    coverage run --source=z3c.testing -m zope.testrunner --test-path=src
```

Comment:
Good catch. I seem to have copied it without thinking about it. Source is already defined in `.coveragerc`.